.github/workflows/slither.yaml

```
@@ -34,7 +36,7 @@
       - name: Install Node.js
         uses: actions/setup-node@v2
```

actionlint (1.7.7)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)